src/main/resources/application.properties

```diff
@@ -1,5 +1,5 @@
-spring.datasource.url=jdbc:h2:mem:testdb
-spring.datasource.driverClassName=org.h2.Driver
-spring.datasource.username=sa
+spring.datasource.url=jdbc:mariadb://localhost:3306/railmetrics
+spring.datasource.username=user
```

Comment:
same, but i edited the application.properties in 5-datenabruf-mit-async-und-scheduled-einrichten so ok for the moment

Reply:
spring.datasource.url=jdbc:mariadb://localhost:3306/railmetrics
spring.datasource.username=user
spring.datasource.password=password
spring.datasource.driver-class-name=org.mariadb.jdbc.Driver
spring.jpa.hibernate.ddl-auto=update

this is the newest version, i dk why its in the old one